harness/determined/pytorch/samplers.py

```diff
@@ -96,9 +94,12 @@ class DistributedBatchSampler(torch.utils.data.BatchSampler):
     def __init__(
         self, batch_sampler: torch.utils.data.BatchSampler, num_workers: int, rank: int
     ) -> None:
-        check.gt(rank, -1, "rank must be non-negative")
-        check.gt(num_workers, 0, "num_workers must be positive")
-        check.lt(rank, num_workers, "rank must be less than num_workers")
+        if rank < 0:
+            raise ValueError("rank must be non-negative.")
+        if num_workers < 1:
+            raise ValueError("num_workers must be strictly positive.")
```

Comment:
nit: either just "positive" or "greater than zero".  "Strictly" positive is not a term that I think is in common use (I've never seen it on any math problem).  I've always understood "positive" to mean what you mean here.

Reply:
I've definitely encountered that phrase in mathmatics, but as always, YMMV.
Anyway, done!